src/content/administration/dashboard.mdx

```diff
@@ -70,3 +70,42 @@ For each namespace, an administrator will be able to:
 The Secrets section of the Admin dashboard lets you define cluster-wide secrets. Using the `$SECRET_NAME` syntax, you can refer to these secrets directly in your Okteto manifest. They are also injected into the [environment running](/docs/cloud/okteto-cli/#built-in-tools-when-deploying-to-okteto-cloud) the commands in the `deploy` section of the manifest.
 
 > Secrets [defined by the user](/docs/cloud/secrets/) take precedence over the secrets defined here in case of a name collision.
+
+To inject these secrets into your Dockerfile, we recomment to use build secrets.
+
+### Build Secrets Sample
+
+Build secrets is the recommended way to inject secrets into your Dockerfile build process.
```

Comment:
This is where I struggle de most. Because they also work with the open source. Another option is an entry in the community and link it from the admin secrets, user secrets, and the build secrets docs

Reply:
No, this should be an official doc, is not an edge case or something like that.

Reply:
- [ ] We can rename Build Service to Build and them have multiple sections there for this, to explain the service, etc. and we can have entries in community for more specific of complex examples